addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -84,15 +84,9 @@ def _check_before_xml_exporting(self):
             raise UserError(_("%s must have a country") % (seller.display_name))
 
         # <1.1.1.2>
-        if not seller.vat:
-            raise UserError(_("%s must have a VAT number") % (seller.display_name))
-        elif len(seller.vat) > 30:
+        if seller.vat and len(seller.vat) > 30:
```

Comment:
It was not meant to just remove those checks.  They are here for a reason. You should check if the country of the partner is not False and not Italian to be able to skip them!